core/src/test/java/io/kestra/core/runners/RestartTest.java

```diff
@@ -17,7 +18,7 @@ void restartFailedThenFailureWithGlobalErrors() throws Exception {
         restartCaseTest.restartFailedThenFailureWithGlobalErrors();
     }
 
-    @Test
+    @RetryingTest(5)
     void restartFailedThenFailureWithLocalErrors() throws Exception {
         restartCaseTest.restartFailedThenFailureWithLocalErrors();
```

Comment:
I dont think its related to your PR, and its just make a flakky test more difficult to find

Reply:
I saw it fail a lot, so I added this annotation.
We decided to retry flaky tests to be able to move forward, there are already a lot of them.
I agree we should get rid of those annotations at some point and fix the test (or the feature under test) but we need reliable CI.